lib/mongo.rb

```diff
@@ -75,8 +75,16 @@
 require 'mongo/version'
 require 'mongo/write_concern'
 require 'mongo/utils'
+require 'mongo/config'
 
 module Mongo
+  extend Forwardable
+  extend self
```

Comment:
Instead of `extend self` I prefer `class << self` syntax so that the methods are defined in the one scope where they are supposed to be used from instead of two scopes.